### PR TITLE
Issue/mod tools UI

### PR DESCRIPTION
**Pull request by tusharravindran**
Banned User Controller Resolved

1. Only Ban Users Joined in Community 
2. Do not Ban Self /Moderator 
3. Do not Ban Same User Again

### CODE REVIEW

File: app/controllers/banned_users_controller.rb [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
+            flash[:alert] = t(:not_present_to_ban, username: @username.username)
+        end 
     end
-
-    def banned_user_params
-     params.require(:banned_user).permit(:username, :reason, :explanation, :account_id, :community_id)
-    end
-
+    redirect_to mod_path(@community) and return
+end
```
Comment (iMagesh) [inline app/controllers/banned_users_controller.rb:27]: there seems to be too many if and else here.. can we refactor this?

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!-- <div class="d-flex flex-column flex-md-row align-items-center p-5 px-md-3 mb-3 bg-white border-bottom shadow-sm">
+<%# <!-- <div class="d-flex flex-column flex-md-row align-items-center p-5 px-md-3 mb-3 bg-white border-bottom shadow-sm">
```
Comment (iMagesh) [inline app/views/layouts/_footer.html.erb:1]: remove comments before committing code

File: app/controllers/banned_users_controller.rb (hunk 1 above)
Comment (tusharravindran) [reply app/controllers/banned_users_controller.rb]: resolved and refactored code with less if else

File: app/views/communities/_ban_users.html.erb [hunk 3]
```diff
@@ -41,3 +41,13 @@
...
+
+
+
+
+
+
+
+
+
+
```
[the remark this answers is not in the document]
Comment (tusharravindran) [reply app/views/communities/_ban_users.html.erb]: code refactored and comment resolved!

File: app/views/layouts/_footer.html.erb (hunk 2 above)
Comment (tusharravindran) [reply app/views/layouts/_footer.html.erb]: code refactored and comment resolved!

File: app/controllers/banned_users_controller.rb [hunk 4]
```diff
@@ -1,33 +1,23 @@
...
+    redirect_to mod_path(@community) and return flash[:notice] = t(:admin_ban, username: @username.username) if @username == current_account
+    @subscriptions = Subscription.where(account_id: @username.id, community_id: params[:banned_user][:community_id])
+    redirect_to mod_path(@community) and return flash[:alert] = t(:not_present_to_ban, username: @username.username) unless @subscriptions.any?
+    @banned_user.account_id = @username.id  
+    if BannedUser.find_by(account_id: @banned_user.account_id, community_id: @banned_user.community_id).nil? 
+      flash[:notice] = t(:banned_successfully, username: @username.username)
+      @banned_user.save
+    else
+      flash[:alert] = t(:already_banned, username: @username.username)
+    end
```
[the remark this answers is not in the document]
Comment (tusharravindran) [reply app/controllers/banned_users_controller.rb]: refactored and resolved using before action callbacks

**Pull request merged**